lib/Activity.py

```diff
@@ -189,25 +208,48 @@ def __str__(self):
         activity_name = self.name
         if not self.api.activity_exists(activity_name):
             activity_name += " [local only]"
-
-        retstring = "+" + "-" * (table_width - 2) + "+\n"
-        retstring += f"| Activity: {activity_name:<{table_width - 14}} |\n"
-        retstring += tstring
+        retstring = []
```

Comment:
This change isn't really necessary.  I was hoping to make the string formatting a little simpler.  We could revert it if anyone prefers the other way.